src/store/region-context.tsx

```diff
@@ -58,7 +58,7 @@ export const RegionProvider = ({ children }: RegionProviderProps) => {
         const is_eu_residence = eu_countries.includes(residence)
         const is_eu_location = is_eu_residence || (!residence && is_eu_country_ip)
         const is_eu = is_eu_location || isEuDomain()
-        const is_non_eu = !is_eu
+        const is_non_eu = !is_eu && !is_africa
```

Comment:
Let's not modify this `is_non_eu` create a new variable to use for the footer. Also I think the change is not for all africs, it should be for 3 countries
```suggestion
        const is_non_eu = !is_eu && !is_africa
```

Reply:
The countires are
`South Africa, Botswana, Ecuador`

Reply:
Good point 👍  I knew that this countries are part of new `cpa plan`. In this case we can easy create new variables `cpa_plan_countries` & 'is_cpa_plan'